instaloader/__main__.py

```diff
@@ -66,6 +67,45 @@ def filterfunc(item) -> bool:
 
     return filterfunc
 
+def get_cookies_from_instagram(domain, browser, cookiefile="", cookieName=''):
+    supported_browsers = {
+        "chrome": browser_cookie3.chrome,
+        "firefox": browser_cookie3.firefox,
+        "edge": browser_cookie3.edge,
+        "brave": browser_cookie3.brave,
+        "opera": browser_cookie3.opera,
+        "safari": browser_cookie3.safari
+    }
+
+    if browser not in supported_browsers:
+        print("Loading cookies from the specified browser failed. Supported browsers are Chrome, Firefox, Edge, Brave, Opera and Safari")
+        return {}
+
+    Cookies = {}
+    browserCookies = list(supported_browsers[browser](cookie_file=cookiefile))
```

Comment:
To be consistent with how variables are named, it should rather be `cookies` and `browser_cookies`.

However I do not well understand the purpose of the `Cookies` variable. Is it just to filter the `browserCookies` that have the `domain` attribute set?

Reply:
Yes, `browser_cookies` returns a list of cookies which we need to format in order to set cookies with ease this is done using `instaloader.context._session.cookies.update(cookie)` 

In this case cookie is a key value pair of cookie name and value that was **filtered and formatted** in the `get_cookies_from_instagram` function.

Reply:
I have fixed the naming conventions